test/Error/error_cause.js

```diff
@@ -0,0 +1,64 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+
+const o = {};
+const tests = [
+    {
+        name: "Error.prototype.cause",
+        body: function () {
+            assert.isFalse("cause" in Error.prototype, "Cause property must not exist in *Error.prototype");
+        }
+    },
+    {
+        name: "Error().cause === undefined",

```

Comment:
Should also have a case where we call the constructor with 1 parameter and check it has a message but not a cause.